src/ngMock/angular-mocks.js

```diff
@@ -2356,15 +2356,10 @@ angular.mock.$ComponentControllerProvider = ['$compileProvider',
  * @packageName angular-mocks
  * @description
  *
- * # ngMock
- *
  * The `ngMock` module provides support to inject and mock AngularJS services into unit tests.
  * In addition, ngMock also extends various core AngularJS services such that they can be
  * inspected and controlled in a synchronous manner within test code.
  *
- *
- * <div doc-module-components="ngMock"></div>
```

Comment:
Are these no longer needed?

Reply:
No, I think they are left over from a time when these were generated differently. The directive doesn't exist anymore. dgeni renders the components during the docs build.